docs/guide/installation-procedure.md

```diff
@@ -44,7 +44,7 @@ sudo su -
 First of all, Webots should be authenticated with the [Cyberbotics.asc](https://www.cyberbotics.com/Cyberbotics.asc) signature file which can be downloaded from the [Webots download page](https://www.cyberbotics.com/download), and installed using this command:
 
 ```sh
-apt-key add /path/to/Cyberbotics.asc
+curl -s -L https://www.cyberbotics.com/Cyberbotics.asc | apt-key add -
```

Comment:
Thank for your this nice suggestion. It would be even better with a sudo:
```suggestion
curl -s -L https://www.cyberbotics.com/Cyberbotics.asc | sudo apt-key add -
```

Reply:
If you agree with my `sudo` addition, can you please commit this change (as I cannot do it on your repo), so that I can merge your PR as is?